ERCS/erc-new.md

```diff
@@ -0,0 +1,135 @@
+---
```

Comment:
```suggestion
---
eip: 7764
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.